### PR TITLE
[vsphere] rename bastion leased var name

### PR DESCRIPTION
**Pull request by jianlinliu**
1. Rename bastion lease var name to `VSPHERE_BASTION_LEASED_RESOURCE` from `VSPHERE_CONNECTED_LEASED_RESOURCE` so that be easy to understand the lease usage.
2. Rename extra lease var for workers to `VSPHERE_EXTRA_LEASED_RESOURCE` in `vsphere-ipi-remote-worker` and `vsphere-ipi-zones-multisubnets-external-lb` cases.

### CODE REVIEW

**Comment by jianlinliu** [comment]
cc @sgaoshang @wewang58 @jinyunma to review.

File: ci-operator/step-registry/vsphere/remote-worker/vsphere-remote-worker-commands.sh [hunk 1]
```diff
@@ -8,10 +8,10 @@ trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wa
 
 remote_machineset_name="remote-worker"
 remote_worker_number="${REMOTEWORKER_NUMBER}"
-declare vsphere_connected_portgroup
+declare vsphere_bastion_portgroup
```
Comment (sgaoshang) [inline ci-operator/step-registry/vsphere/remote-worker/vsphere-remote-worker-commands.sh:11]: other lgtm, only this is a little odd for remote worker workflow

Comment (sgaoshang) [reply ci-operator/step-registry/vsphere/remote-worker/vsphere-remote-worker-commands.sh:11]: or we may need to separate it for remote worker

Comment (jianlinliu) [reply ci-operator/step-registry/vsphere/remote-worker/vsphere-remote-worker-commands.sh:11]: How to introduce a new var name for remote-worker case? e.g: VSPHERE_EXTRA_LEASED_RESOURCE, once that, I think `VSPHERE_MULTIZONE_LEASED_RESOURCE` can be replaced with the the same var.  Even more, actually bastion segment is also kind of an extra segment, we also can merge it, but to be safer, maybe we can keep `VSPHERE_BASTION_LEASED_RESOURCE`.

Comment (sgaoshang) [reply ci-operator/step-registry/vsphere/remote-worker/vsphere-remote-worker-commands.sh:11]: Both OK from me, actually I changed "VSPHERE_BASTION_LEASED_RESOURCE" to "VSPHERE_CONNECTED_LEASED_RESOURCE" in order to adapt remote worker.

**Comment by jianlinliu** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-vsphere-ipi-remote-worker-f14

**Comment by jianlinliu** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-vsphere-ipi-zones-multisubnets-external-lb-f28

**Comment by jianlinliu** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-vsphere-ipi-zones-multisubnets-external-lb-f28

**Comment by jianlinliu** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-vsphere-ipi-remote-worker-f14

**Comment by jianlinliu** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-vsphere-ipi-remote-worker-f14

**Comment by jianlinliu** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-vsphere-ipi-zones-multisubnets-external-lb-f28

**Comment by jianlinliu** [comment]
@sgaoshang @WenXinWei @jinyunma the pj-rehearse job get passed on installation, ready to merge.

**Comment by wewang58** [comment]
vsphere-ipi-zones-multisubnets-external-lb-f28-ipi-conf-vsphere-zones-multisubnets and vsphere-ipi-remote-worker-f14-vsphere-remote-worker succeed in jobs so it's ok for me

**Comment by jianlinliu** [comment]
cc @liangxia to review.

**Pull request merged**